README.md

```diff
@@ -24,6 +24,12 @@ Take your conversations with you and never miss the action with our <br />iOS mo
 
 [About Rocket.Chat](https://github.com/RocketChat/Rocket.Chat/#about-rocketchat)
 
+# Beta Access (TestFlight)
+
+You can signup to our TestFlight builds by acessing this link: https://testflight.apple.com/join/3gcYeoMr.
+
+We've also a channel in our developer community, you can [access here](https://open.rocket.chat/channel/ios-beta).
```

Comment:
*we also have OR

And maybe we should explicitly mention #ios-beta in the text, so that people can search for it, if they want to. 🤷🏻‍♂️